appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -1477,7 +1475,16 @@ Blockly.ReplMgr.getFromRendezvous = function() {
                 rs.webrtc = json.webrtc === "true";
                 rs.useproxy = json.useproxy === "true";
                 rs.hasfetchassets = rs.android || rs.webrtc;
-
+                if (!(rs.android) && Blockly.ReplMgr.hasExtensions()) {
+     var content = "Extentions are not currently supported for IOS Devices";
```

Comment:
We should put the message in Blockly.Msg so that it can be internationalized. See blocklyeditor/src/msg/ai_blockly/messages.json.